### DISCUSSION TITLE
Info On Form Submissions

### DISCUSSION THREAD

**Discussion by MisterRich** [MasaCMS/MasaCMS#162] [General]
Hopefully someone can point me where to even start looking for these issues.  When a form is submitted what processes does it go through?  Somewhere there is something that manipulates the text fields, not sure if it is a validation, a form protectection or a conversion but I now have 2 examples that are making be a bit crazy.  You can test it yourself by putting either of these text  in as the title of a page.  First example is 2E25136 gets changed to "∞", the second example is houseofstring.com the "string."  portion is changed to [INVALID] This one is a bigger issue as it prevents that from being a user's email address domain. Any ideas or thoughts on how to stop this would be appreciated. Thanks!  FYI This happens in Mura 7.x and Masa 7.3, 7.4

**Comment by guustnieuwenhuis** [comment]
I would test using the events on forms to test: https://docs.masacms.com/customizing-extending-masa/events/event-hooks/#form-specific-events
